rose_planner_node:
  ros__parameters:
    odom_topic: /Odometry
    goal_topic: /rose_goal
    target_frame: map
    robot_size: [0.2, 0.2]
    use_control_output: true
    path_search:
      a_star: 
        clearance_weight: 2.0
        obstacle_penalty_weight: 10.0
        heuristic_weight: 1.0
      resampler:
        max_vel: 4.0
        acc: 3.0
        dt: 0.1
    trajectory_opt:
      smooth_weight: 3.0
      obstacle_weight: 5000.0
      time_weight: 10.0
    mpc_control:
      dt: 0.01
      max_iter: 20
      predict_steps: 100
      max_speed: 3.0
      min_speed: 0.2
      max_accel: 2.0
      delay_num: 0
      Q: [150.0,150.0,0.1,0.1]
      Rd: [1.0,0.1]
      R: [0.1,0.1]
    rose_map:
      log_time: true
      pointcloud_topic: /cloud_registered
      target_frame: map
      sensor_frame: front_mid360
      max_update_rate: 10
      occ_map:
        origin: [0.0, 0.0, 0.0]
        size: [17.0, 17.0, 1.0]
        voxel_size: 0.02
        log_hit: 0.85
        log_free: -0.3
        log_min: -5.0
        log_max: 10.0
        occ_th: 0.0       
        min_shift: 2
        timeout: 1.5
        unknown_is_occupied: false
        use_ray: true
        max_ray_range: 20.0
      acc_map:
        origin: [0.0, 0.0]
        size: [12.0, 12.0]
        voxel_size: 0.05
        origin2base: 0.2
        min_diff_z: 0.1
        min_block_count: 2
        block_ratio: 0.15
        kernel_size: 3
        dilate_iter: 1
        use_static_map: true
        static_map_path: "/home/hy/wust_nav2/src/bringup_nav/map/simulation/rmuc_2026.yaml"
      esdf:
        use_diagonals: true